app/lib/admin/actions/remove_package_from_publisher.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:clock/clock.dart';
+import 'package:pub_dev/admin/actions/actions.dart';
+
+import '../../package/backend.dart';
+import '../../package/models.dart';
+import '../../publisher/backend.dart';
+import '../../shared/datastore.dart';
+
+final removePackageFromPublisher = AdminAction(
+    name: 'remove-package-from-publisher',
+    options: {
+      'package': 'name of the package to remove from its current publisher'
+    },
+    summary:
+        'Removes <package> from its current publisher, and makes that publisher\'s owners uploaders',
+    description: '''
+Removes <package> from its current publisher, and makes that publisher\'s owners
+ uploaders.
+
+Fails if that package currently has no uploader.
```

Comment:
This doesn't make sense, should it be negated?

Reply:
Good catch, should be publisher